plugins/queue/rabbitmq_amqplib.js

```diff
@@ -39,7 +40,7 @@ exports.init_amqp_connection = function () {
     var autoDelete = cfg.autoDelete === "true" || false;
     deliveryMode = cfg.deliveryMode || 2;
 
-    amqp.connect("amqp://"+user+":"+password+"@"+host+":"+port, function (err, conn){
+    amqp.connect("amqp://"+user+":"+password+"@"+host+":"+port+vhost, function (err, conn){
```

Comment:
In looking at the cited URL, it appears the vhost name is appended after the port with a `/` delimiter. I don't see that getting added anywhere.